compiler/x/codegen/OMRTreeEvaluator.cpp

```diff
@@ -6925,12 +6926,14 @@ OMR::X86::TreeEvaluator::arrayToVectorMaskHelper(TR::Node *node, TR::CodeGenerat
          shiftAmount = 31;
          expandOp = TR::InstOpCode::PMOVZXBDRegReg;
          v2mOp = TR::InstOpCode::VPMOVD2MRegReg;
+         integralType = TR::Int32;
          break;
       case TR::Int64:
       case TR::Double:
          shiftAmount = 63;
          expandOp = TR::InstOpCode::PMOVZXBQRegReg;
          v2mOp = TR::InstOpCode::VPMOVQ2MRegReg;
+         integralType = TR::Int64;
```

Comment:
May I ask you to add a comment to these two assignments to `integralType` &mdash; or at its declaration &mdash; indicating why this is needed?  It’s obvious in the context of this commit, but it might not be so obvious in a few years to someone who is new to this code.

Reply:
See [force-push](https://github.com/eclipse-omr/omr/compare/6c338239bbfd350f469a0f719efb13b1f8fc7454..6b655c1a7fe19099b0dc8dff49583b0c3a05ff73)